clusterloader2/pkg/prometheus/prometheus.go

```diff
@@ -39,15 +39,19 @@ import (
 )
 
 const (
-	namespace                    = "monitoring"
-	storageClass                 = "ssd"
-	coreManifests                = "$GOPATH/src/k8s.io/perf-tests/clusterloader2/pkg/prometheus/manifests/*.yaml"
-	defaultServiceMonitors       = "$GOPATH/src/k8s.io/perf-tests/clusterloader2/pkg/prometheus/manifests/default/*.yaml"
-	masterIPServiceMonitors      = "$GOPATH/src/k8s.io/perf-tests/clusterloader2/pkg/prometheus/manifests/master-ip/*.yaml"
-	checkPrometheusReadyInterval = 30 * time.Second
-	checkPrometheusReadyTimeout  = 15 * time.Minute
-	numK8sClients                = 1
-	nodeExporterPod              = "$GOPATH/src/k8s.io/perf-tests/clusterloader2/pkg/prometheus/manifests/exporters/node-exporter.yaml"
+	namespace                     = "monitoring"
+	storageClass                  = "ssd"
+	checkPrometheusReadyInterval  = 30 * time.Second
+	checkPrometheusReadyTimeout   = 15 * time.Minute
+	numK8sClients                 = 1
+	defaultPrometheusManifestPath = "$GOPATH/src/k8s.io/perf-tests/clusterloader2/pkg/prometheus/manifests"
+)
+
+var (
+	coreManifests           string
```

Comment:
TBH - I'm not a fan of global variables.

Instead, we should add fields to PrometheusConfig and fill them in CompleteConfig.

Reply:
@wojtek-t  Thanks! I fixed the bug and moved the global config parameters to PrometheusConfig. The const "DefaultPrometheusPath" was removed too and use the pflag default value instead. Please review it.